src/App.js

```diff
@@ -2,28 +2,26 @@ import React from 'react';
 
 import './App.css';
 
-import posts from './api/posts';
-import comments from './api/comments';
-import users from './api/users';
+import postsList from './api/posts';
+import commentsList from './api/comments';
+import usersList from './api/users';
+import PostList from './api/PostList';
+
+function postUserComment(posts, users, comments) {
+  return posts.map(item => (
+    {
+      ...item,
+      user: users.find(person => person.id === item.userId),
+      comment: comments.filter(elem => elem.postId === item.id),
```

Comment:
Тут будет не один коммент, а 5
```suggestion
      comments: comments.filter(elem => elem.postId === item.id),
```

Reply:
Понял